api/src/main/java/io/opentelemetry/correlationcontext/CorrelationContext.java

```diff
@@ -74,6 +75,28 @@
      */
     Builder setParent(CorrelationContext parent);
 
+    /**
+     * Sets the parent {@link CorrelationContext} to use from the specified {@code Context}. If no
+     * parent is provided, the value of {@link CorrelationContextManager#getCurrentContext()} at
```

Comment:
maybe "If no parent context is available in the provided Context, the value of ...."

Reply:
@carlosalberto thoughts on my suggestion?